amazon-kinesis-client/src/main/java/software/amazon/kinesis/leases/LeaseManagementConfig.java

```diff
@@ -220,6 +220,15 @@
     private InitialPositionInStreamExtended initialPositionInStream =
             InitialPositionInStreamExtended.newInitialPosition(InitialPositionInStream.TRIM_HORIZON);
 
+    /**
+     * The position for getting records from an "orphaned" stream that is in the lease table but not tracked
+     * Default assumes that the stream no longer need to be tracked, so use LATEST for faster shard end.
+     *
+     * <p>Default value: {@link InitialPositionInStream#LATEST}</p>
+     */
+    private InitialPositionInStreamExtended orphanedStreamInitialPositionInStream =
```

Comment:
I was thinking this can be part of the MultiStreamTracker interface. Do you have any concerns?

Reply:
Make sense, done.